src/main/kotlin/no/skatteetaten/aurora/boober/feature/Feature.kt

```diff
@@ -227,7 +227,7 @@ class HeaderHandlers private constructor(defaultAppName: String, defaultEnvName:
                 validator = {
                     it.allowedPattern(
                         "^[a-z][-a-z0-9]{0,38}[a-z0-9]$",
-                        "Name must be alphanumeric and no more than 40 characters",
+                        "Name must be alphanumeric, lowercase and no more than 40 characters",
```

Comment:
Dette er fremdeles ikke helt eksakt... Kravet er jo et gyldig DNS subdomain... Forslag til tekst?

Reply:
Man kan jo ta et valg om man vil eksponere regexen, selv om det ikke er pent.

Ellers så kan man vel skrive:
The first letter of name must be a letter [a-z], the last letter must be alphanumeric. Total length needs to be no more than 40 characters. The rest of the characters can be alphanumeric and hyphen.

Reply:
skjer det noe her @anderius ?

Reply:
@stromland har du noen tanker her?

Reply:
Ser dette kommer opp med jevne mellomrom, senest her https://chat.sits.no/skatt-it/pl/m4eu1pk5k3byzxdzikjajs3dch

Reply:
@kris456 Har et nytt forslag nå, basert på din tekst!